orbstation/orb_modules/flavor_text/examine_procs.dm

```diff
@@ -0,0 +1,44 @@
+// Handles the hyperlink to display the "close examine" box in chat
+/mob/living/carbon/human/Topic(href, href_list)
+	. = ..()
+
+	if(href_list["show_flavor_text"])
+		var/mob/living/carbon/human/target = locate(href_list["show_flavor_text"])
+		if(target) //code here copied from "run_examinate()" in mob.dm. might be ugly.
+			var/list/result = target.examine_more(usr)
+			if(result.len) //add proper linebreaks
+				for(var/i in 1 to (length(result) - 1))
+					result[i] += "\n"
+			to_chat(usr, examine_block("<span class='infoplain'>[result.Join()]</span>"))
+
+// Handles the hyperlink to display the "close examine" box in chat (silicon this time)
+/mob/living/silicon/robot/Topic(href, href_list)
+	. = ..()
+
+	if(href_list["show_flavor_text"])
```

Comment:
Can `"show_flavor_text"` be a define rather than a hardcoded string? 🙂

Reply:
It probably could, yes - but all other href implementations in the code do NOT use defines for whatever reason, so I did it the same way.